tests/bug-2917/test.rules

```diff
@@ -0,0 +1,15 @@
+alert http $HOME_NET any -> $EXTERNAL_NET $HTTP_PORTS (msg:"TEST 1"; flow:to_server,established; content:"GET"; http_method; flowbits:set,test; sid:1;)
+
+# does not work - based on the original rules
```

Comment:
what does not work?

Reply:
This is where, before `227d1d967a5902` (that suppressed the error), one would get the error `Unable to find the sm in any of the sm lists`.
The error disappeared if things were done differently (in 3 different ways) in the following 3 rules.

Reply:
can you update the message to reflect that in the past this generated an error/warning?